spec/data_services_api/dataset_spec.rb

```diff
@@ -5,7 +5,7 @@
 describe 'DataServiceApi::Dataset' do
   before do
     VCR.insert_cassette name, record: :new_episodes
-    @dataset = DataServicesApi::Service.new(url: 'http://localhost:8080').dataset('ukhpi')
+    @dataset = DataServicesApi::Service.new(url: 'http://localhost:8888').dataset('ukhpi')
```

Comment:
This should be configurable, or there should be instructions in the readme as to what to set up on `localhost:8888`

Reply:
What should the readme contain? A link to the new SAPI API for LR?

Reply:
The key test is whether someone who doesn't have the benefit of already having worked on the project (or did, but six months ago and has now forgotten!) can recreate a working setup using just these instructions. As it stands, I think the answer would be "no". They would have to ask someone how to do some of the steps.

A general principle for these kinds of situation is to not hard-code the URL into the test, but instead put it into an environment variable. Then, you can say in the readme what variables to set in the environment, what the default values are, and, if needed, how to set up your dev system so that the tests will work